modules/enabling-scanner-v4-after-operator-installation.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-operator-installation_{context}"]
+= Enabling RHACS Scanner V4 after Operator installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default. If you did not enable it during installation, you can enable it after installation.
+
+First, you must enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. In the console, select *Operators* -> *Installed Operators* and select the {product-title-short} Operator.
+. Click *Central* in the menu bar.
+. Click *stackrox-central-services* to view the *stackrox-central-services* page, if you installed {product-title-short} using the default values.
+. Click the name of the cluster where Central was installed. The default value is *stackrox-central-services*.
+. Select the YAML tab.
+. Edit the YAML file as shown in the following example:
++
+[source,yaml]
+----
+  scannerV4:
+    scannerComponent: Enabled
+----
+
+After you enable Scanner V4 on the cluster where Central is installed, you can enable Scanner V4 on the secured cluster.
+
+[NOTE]
+====
+You must have previously configured your init bundle or cluster registration secret (CRS) to allow communication between Central and the secured cluster.
+====
+
+.Procedure
+
+. In the console, select *Operators* -> *Installed Operators* and select the {product-title-short} Operator.
+. Click *Secured Cluster* in the menu bar.
+. Click *stackrox-secured-cluster-services* to view the *stackrox-secured-cluster-services* page, if you installed {product-title-short} using the default values.
```

Comment:
Same comment as I made earlier in this module about whether this is a condition and should be rewritten:
```suggestion
. If you installed {product-title-short} using the default values, click *stackrox-secured-cluster-services* to view the *stackrox-secured-cluster-services* page.
```